samples/dect/dect_phy/dect_shell/src/dect/mac/dect_phy_mac_shell.c

```diff
@@ -431,21 +431,29 @@ static int dect_phy_mac_dissociate_cmd(const struct shell *shell, size_t argc, c
 /**************************************************************************************************/
 
 static const char dect_phy_mac_rach_tx_cmd_usage_str[] =
-	"Usage: dect mac rach_tx <c_data> [<options>]\n"
+	"Usage: dect mac rach_tx [stop] | -d <data> [<options>]\n"
 	"Options:\n"
 	"  -d, --data <data_str>,  Data to be sent.\n"
 	"  -p, --tx_pwr <integer>, TX power (dBm) (default 0 dBm)\n"
 	"  -m, --tx_mcs <integer>, TX MCS (integer). Default: 0.\n"
 	"  -t, --long_rd_id <id>,  Target long rd id (default: 38).\n"
```

Comment:
To make these consistent:
```suggestion
	"  -p, --tx_pwr <integer>, TX power (dBm). Default: 0 dBm.\n"
	"  -m, --tx_mcs <integer>, TX MCS (integer). Default: 0.\n"
	"  -t, --long_rd_id <id>,  Target long rd id. Default: 38.\n"
```